### PR TITLE
Header

### PR DESCRIPTION
**Pull request by Yvonne-Wong**
Added headers to Home, Rover selection, Curiosity and News pages

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "plugins": [
         "react",
         "@typescript-eslint"
+
```
Comment (ChristianMcc) [inline .eslintrc.json]: can delete this new line

File: src/Home/Home.tsx [hunk 2]
```diff
@@ -4,12 +4,18 @@ import { PicOfTheDay } from './PicOfTheDay/PicOfTheDay';
...
+
+
 
 function Home() {
   return (
     <div>
       <MobileNavbar />
-      <h1>MotherBase</h1>
+      <Header 
+      imgSource = {`${process.env.PUBLIC_URL}/images/motherbase.png`}
```
Comment (ChristianMcc) [inline src/Home/Home.tsx:16]: I can't say I'm the biggest fan of this workaround but let's go with it for now 😂

**Review by ChristianMcc — COMMENTED** [review]
looking good! couple of questions/comments

File: src/Header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './Header.scss'
+
+interface HeaderName {
+    imgSource: string,
+    altTag: string,
+}
+
+function Header(props: HeaderName) {
+    return<div className='headerContainer'>
```
Comment (ChristianMcc) [inline src/Header/Header.tsx:10]: I think I was imagining the navbar to be in this component, was this not done due to how Emily and Mariam have tried to blur that background of the page when the navbar is open or for another reason? Happy o leave it for now if it's not a simple case of just moving it into this component

Comment (Yvonne-Wong) [reply src/Header/Header.tsx:10]: Yes Emily Mariam specified that navBar had to be the top component for the blurring to work correctly

**Review by ChristianMcc — APPROVED** [review]
Once you've deleted the extra new line in eslintrc.json feel free to merge :)

File: .eslintrc.json (hunk 1 above)
Comment (Yvonne-Wong) [reply .eslintrc.json]: This has been actioned :)

**Pull request merged**